packages/back-end/features/step-definitions/then.ts

```diff
@@ -44,3 +46,63 @@ Then(
     expect(persons[0].properties.name).to.equal(name);
   }
 );
+
+Then(
+  '{string} is saved in the database',
```

Comment:
Hard to see if this refer to a user, a person, something else
```suggestion
  'the person {string} is saved in the database',
```